### PR TITLE
New question on hidden metadata.

### LINKED ISSUE #130: 'Hidden' data question needed?
We have a continuing problem with propagation of normally-invisible but easily revealed data. The poster-child is EXiF data in images, which can be very revealing but often not exposed. But there are other examples, and many formats support normally-hidden data (e.g. in text formats that are interpreted, metadata and comments are supported, including in HTML). 

If you are defining a format, does it enable the carriage of data that is not normally revealed, but that can and sometimes does carry information about a person? Examples include metadata fields and comment blocks in text formats (sometimes used for comments like "last edited by Brutus on 15 March 55BC"), but can also include e.g. client-side cropping of images, client-side trimming the time-intervals of audio and video played, and so on.

### PR DESCRIPTION
**Pull request by hober**
Fixes #130.

***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/security-questionnaire/pull/138.html" title="Last updated on Jul 17, 2024, 8:39 PM UTC (f0a410f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/security-questionnaire/138/db1ac20...f0a410f.html" title="Last updated on Jul 17, 2024, 8:39 PM UTC (f0a410f)">Diff</a>

### CODE REVIEW

**Comment by hober** [comment]
@dwsinger, please take a look?

File: index.bs [hunk 1]
```diff
@@ -313,6 +313,35 @@ See also
...
 
+<h3 class=question id=hidden-data>
+  Does data exposed by your specification carry related but distinct
+  information that may not be obvious to users?
+</h3>
+
+Features which enable users
+to share data with origins
+should ensure that such data
+does not carry related but distinct metadata
```
Comment (dwsinger) [inline index.bs:324]: Maybe whether it's related or not is not the point. "does not carry embedded, possibly hidden, information that the user is unaware is being relayed, and might not consent to being relayed if they were aware"?

File: index.bs [hunk 2]
```diff
@@ -313,6 +313,35 @@ See also
...
+often contain metadata about
+where and when the image, video, or audio was captured
+and
+what kind of device captured or produced the data.
+When uploaded,
+this kind of metadata
+may reveal to origins
+information the user did not intend to reveal,
+such as the user's location
+and socioeconomic status.
```
Comment (dwsinger) [inline index.bs:338]: Not sure that socioeconomic status is the top worry ("an expensive camera was used here"?). But in the presence of systems that do object, and face detection; and object, location, and face recognition, it's not a stretch to realize that the metadata includes such analytic information.

File: index.bs [hunk 3]
```diff
@@ -313,6 +313,35 @@ See also
...
+such as image or video files
+often contain metadata about
+where and when the image, video, or audio was captured
+and
+what kind of device captured or produced the data.
+When uploaded,
+this kind of metadata
+may reveal to origins
+information the user did not intend to reveal,
+such as the user's location
```
Comment (dwsinger) [inline index.bs:337]: and when they were in that location

**Review by dwsinger — CHANGES_REQUESTED** [review]
some minor suggested changes, but looks good overall

**Review by Zeroplot — APPROVED** [review]

File: index.bs (hunk 1 above)
Comment (hober) [reply index.bs]: Tweaked the wording based on this feedback.

File: index.bs (hunk 3 above)
Comment (hober) [reply index.bs]: Good point. Tweaked.

**Review by pes10k — APPROVED** [review]
lgtm

**Comment by Zeroplot** [comment]
Screenshot_20240915-195730

**Pull request merged**